src/device.rs

```diff
@@ -766,6 +796,19 @@ map_enum!(
     Min,
     Max
 );
+map_enum!(
```

Comment:
In other parts of gfx/wgpu ecosystem, we tend to have a specific module (like `conv.rs`) to convert things. Maybe it is useful here too, so that we can make `device.rs` lighter. Up to you :)